packages/block-editor/src/components/block-list/use-block-props/use-scroll-into-view.js

```diff
@@ -10,7 +10,7 @@ export function useScrollIntoView( { isSelected } ) {
 			if ( isSelected ) {
 				const { ownerDocument } = node;
 				const { defaultView } = ownerDocument;
-				if ( ! defaultView.IntersectionObserver ) {
+				if ( ! defaultView?.IntersectionObserver ) {
```

Comment:
What causes `defaultView` to be null?

Reply:
There's some more discussion over here - https://github.com/WordPress/gutenberg/issues/61941#issuecomment-2136534664.

Not entirely sure what causes it, but my best guess is it's a side effect of the page unload, maybe the DOM elements being detached from the DOM.

Not really sure if it's something we need to fix. I just put the PR to demonstrate that the error goes away after this change.